src/table/head/table-head.component.ts

```diff
@@ -171,7 +171,9 @@ export class TableHead implements AfterViewInit {
 	constructor(protected i18n: I18n) {}
 
 	ngAfterViewInit() {
-		this.scrollbarWidth = getScrollbarWidth();
+		setTimeout(() => {
+			this.scrollbarWidth = getScrollbarWidth();
+		});
```

Comment:
setTimeout usually should be avoided but if you think it's really necessary I'll approve

Reply:
In this case we're using `setTimeout` to hook into the Angular change detection, and schedule an update for the next tick ... this usually implies an error in how the values are being propagated, though in this case that may not be true